src/Files.App/UserControls/Widgets/FileTagsWidget.xaml

```diff
@@ -108,24 +109,30 @@
 
 						<controls:AdaptiveGridView.ItemTemplate>
 							<DataTemplate x:DataType="vm:FileTagsItemViewModel">
-								<StackPanel
+								<Grid
```

Comment:
We did we change it to Grid?

Reply:
StackPanel doesn't dynamically adjust column widths, and outputs text into invisible areas without trimming it. So I changed it to Grid.